tests/components/scene/inspector.test.js

```diff
@@ -0,0 +1,42 @@
+/* global assert, process, setup, suite, teardown, test */
+
+suite('inspector', function () {
+  setup(function () {
+    var el = this.sceneEl = document.createElement('a-scene');
+    el.setAttribute('inspector', '');
+    document.body.appendChild(el);
+  });
+
+  teardown(function () {
+    var el = this.sceneEl;
+    el.parentNode.removeChild(el);
+  });
+
+  test('adds inspector SCRIPT element to page', function (done) {
+    var el = this.sceneEl;
+
+    el.addEventListener('loaded', function () {
+      var i;
+      var matchingScripts = 0;
+      var scripts;
+
+      // Have to call method directly because Chrome doesn't provide
+      // a reliable method for KeyboardEvent
+      el.components.inspector.onKeydown({
+        keyCode: 73,
+        ctrlKey: true,
+        altKey: true
+      });
+
+      scripts = document.querySelectorAll('script');
```

Comment:
We could add `data-name="aframe-inspector" aframe-injected` attributes to the injected script tag.